buildbot/master/files/config/environments.py

```diff
@@ -31,6 +31,7 @@ def __add__(self, other):
 
 build_windows = build_common + Environment({
     'CARGO_HOME': '/home/Administrator/.cargo',
+    'HOME': r'C:\buildbot\slave\windows\build',
```

Comment:
this set home directory before bash login, so it not needed to add ```cd``` command before every command.